docs/consume-packages/Package-Source-Mapping.md

```diff
@@ -11,6 +11,11 @@ f1_keywords:
 
 # Package Source Mapping
 
+By default, NuGet will search all package sources when it needs to download a package.
+When a package exists on multiple sources, it may not be deterministic which source the package will be downloaded from.
+With Package Source Mapping, you can filter, per package, which source(s) NuGet will search.
+This ensures that packages you only want to download from a specific source, or a subset of sources, will not accidentally be downloaded from a source considered untrusted for that package id.

```

Comment:
I think the below 2-3 paragraphs might need tweaked so that it doesn't read like we have 2-3 intros.

Please take a look and let me know if you share the same concern.